instrumentation/opentelemetry-instrumentation-grpc/src/opentelemetry/instrumentation/grpc/_server.py

```diff
@@ -113,8 +113,9 @@ def set_trailing_metadata(self, *args, **kwargs):
     def abort(self, code, details):
         self.code = code
         self.details = details
+        self._active_span.set_attribute("rpc.grpc.status_code", code.name)
```

Comment:
I think this is a good change (i.e. `OK` instead of just `0`), but I'm wondering if we should record the number `code.value[0]` as well in another attribute. I'm okay with not recording it, just noting that it was recorded before.

Reply:
Looks like there's still a debate as to whether this should be the numeric code or the text version, the lean is numeric because of language differences.  I implemented this before the last comment there, and so was going to hold off until that PR is accepted first.

Reply:
Is this attribute being added to the spec?

Reply:
@aabmass see https://github.com/open-telemetry/opentelemetry-specification/pull/1156